Tests/run_tests.py

```diff
@@ -97,9 +97,6 @@ def is_pypy():
     'Bio.PDB.PDBIO',
     'Bio.PDB.ResidueDepth',
     'Bio.PDB.vectors',
-    'Bio.phenotype',
-    'Bio.phenotype.parse',
-    'Bio.phenotype.phen_micro',
```

Comment:
Rather than removing them from the general exclude list, you need to move them to the NumPy specific list - they are failing on the TravisCI target which deliberately does not have numpy:

```
...
Bio.pairwise2 docstring test ... ok
Bio.phenotype docstring test ... FAIL, ImportError
Bio.phenotype.phen_micro docstring test ... FAIL, ImportError
Bio.triefind docstring test ... ok
BioSQL docstring test ... ok
BioSQL.BioSeq docstring test ... ok
BioSQL.DBUtils docstring test ... ok
BioSQL.Loader docstring test ... ok
ERROR while importing Bio.phenotype: No module named 'numpy'
ERROR while importing Bio.phenotype.phen_micro: No module named 'numpy'
----------------------------------------------------------------------
```